<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tuan2</class>
 <widget class="QDialog" name="Tuan2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tuan2</string>
  </property>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Lựa chọn bài tập</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bai1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Bài 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bai2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>90</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Bài 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bai3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Bài 3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bai4">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>180</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Bài 4</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
